Tests/SentryTests/SentryCrash/SentryStacktraceBuilderTests.swift

```diff
@@ -70,48 +75,61 @@ class SentryStacktraceBuilderTests: XCTestCase {
         XCTAssertTrue(filteredFrames.count == 1, "The frames must be ordered from caller to callee, or oldest to youngest.")
     }
 
-    func testConcurrentStacktraces() {
-        guard #available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *) else { return }
+    func testConcurrentStacktraces() throws {
+        guard #available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *) else {
+            throw XCTSkip("Not available for earlier platform versions")
+        }
 
         SentrySDK.start { options in
             options.dsn = TestConstants.dsnAsString(username: "SentryStacktraceBuilderTests")
             options.swiftAsyncStacktraces = true
+            options.debug = true
         }
 
         let waitForAsyncToRun = expectation(description: "Wait async functions")
         Task {
+            print("\(Date()) [Sentry] [TEST] running async task...")
             let filteredFrames = await self.firstFrame()
             waitForAsyncToRun.fulfill()
             XCTAssertGreaterThanOrEqual(filteredFrames, 3, "The Stacktrace must include the async callers.")
         }
-        wait(for: [waitForAsyncToRun], timeout: 1)
+
+        wait(for: [waitForAsyncToRun], timeout: 10)
```

Comment:
`m`: Increasing this to 10 looks a bit scary cause it can slow down our test suite. What made you increase this value?

Reply:
looking at the test logs, I saw it taking 5-6 seconds between the log statements that define the boundary completing the async task. no idea why though 🙁

Reply:
we can either special case it for tsan only, or skip these completely, but that seems unwise given what they're supposed to test.

Reply:
Maybe tsan doesn't work well with `backtrace_async`. I would rather special case them for tsan.